### PR TITLE
PMM-7984 Make changes for Submodules FB for UI tests

### PR DESCRIPTION
**Pull request by puneet0191**
With changes in UI tests job to start using Docker-compose, we no longer need aws-staging for checking UI tests for FB's. 

The branch name will be changed back to main after testing FB, also the UI test job name will be changed back

Needs to be merged only after: 
https://github.com/percona/pmm-ui-tests/pull/17
https://github.com/Percona-Lab/jenkins-pipelines/pull/953

When both PR's will be merged will revert temp changes on this one.

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1743-629132c
client docker - perconalab/pmm-client-fb:PR-1743-629132c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1743-629132c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests-temp/454/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1743-629132c
client docker - perconalab/pmm-client-fb:PR-1743-629132c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1743-629132c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/6320/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests-temp/460/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1743-629132c
client docker - perconalab/pmm-client-fb:PR-1743-629132c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1743-629132c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests-temp/467/

File: Jenkinsfile [hunk 1]
```diff
@@ -48,14 +26,12 @@ void runTestSuite(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH
     env.BATS_TESTS_RESULT = testSuiteJob.result
 }
 
-void runUItests(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH, PMM_QA_GIT_COMMIT_HASH, PMM_SERVER_IP) {
-    e2eTestJob = build job: 'pmm2-ui-tests', propagate: false, parameters: [
+void runUItests(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH, PMM_QA_GIT_COMMIT_HASH) {
+    e2eTestJob = build job: 'pmm2-ui-tests-temp', propagate: false, parameters: [
```
Comment (ademidoff) [inline Jenkinsfile]: @puneet0191 Looks good. Do we want to have a job name that suggests this is a permanent change rather than a `temp` solution?

Comment (ademidoff) [reply Jenkinsfile]: I'd suggest we do not keep around pipelines we do not use or no longer use, since they make it difficult to find the right one.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1743-629132c
client docker - perconalab/pmm-client-fb:PR-1743-629132c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1743-629132c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-629132c.tar.gz

File: Jenkinsfile (hunk 1 above)
Comment (puneet0191) [reply Jenkinsfile]: @atymchuk yes, we generally disable pipelines after using them for testing purposes, as you can see in my PR description, I am going to remove the `temp` and change it back, I am only using it for triggering the FB execution.

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests-temp/472/

**Review by yurkovychv — DISMISSED** [review]

**Review by yurkovychv — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1743-4fe2d55
client docker - perconalab/pmm-client-fb:PR-1743-4fe2d55
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-4fe2d55.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1743-4fe2d55&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1743-4fe2d55.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/6988/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/8483/

**Pull request merged**